pkg/server/container_create_windows.go

```diff
@@ -541,6 +541,8 @@ func (c *criService) addOCIMounts(ctx context.Context, g *generator, platform im
 			}
 			src = formattedSource
 			mountType = "virtual-disk"
+		} else if strings.HasPrefix(src, `evd://`) {
```

Comment:
Minor point, is it common practice to use backticks even if there are no escaped characters to ignore? Seems like this routine double quotes are used where there are no back slashes.

Reply:
I think I had used them earlier because I had some special character but then I didn't changed it back. Do you want me to change this to normal quotes?